pkg/k8sutil/stunnel.go

```diff
@@ -21,48 +22,48 @@ func stunnelEnvVars(
 	index int,
 ) []v1.EnvVar {
 
-	springboardListenPort := fmt.Sprintf("%d", 6789 + index)
-	stunnelEnv := []v1.EnvVar {
+	springboardListenPort := fmt.Sprintf("%d", 6789+index)
+	stunnelEnv := []v1.EnvVar{
 		{
-			Name: "STUNNEL_VERIFY_CHAIN",
+			Name:  "STUNNEL_VERIFY_CHAIN",
```

Comment:
why was a space inserted?

Reply:
These formatting changes are thanks to VS Code, not sure why it is doing that.